website/docs/user_guide/rbac.md

```diff
@@ -27,28 +27,43 @@ Unleash comes with a set of built-in roles that you can use. The _global roles_
+| **Editor** | Global | Users with the editor role have access to most features in Unleash but can not manage users and roles in the global scope. Editors will be added as project owners when creating projects and get superuser rights within the context of these projects. Users with the editor role will also get access to most permissions on the default project by default. | All versions |
+| **Viewer** | Global | Users with the viewer role can read global resources in Unleash. | All versions |
+| **Owner** | Project | Users with this the project owner role have full control over the project, and can add and manage other users within the project context; manage feature toggles within the project; and control advanced project features like archiving and deleting the project. | Pro and Enterprise |
+| **Member** | Project | Users with the project member role are allowed to view, create, and update feature toggles within a project, but have limited permissions in regards to managing the project's user access and can not archive or delete the project. | Pro and Enterprise |
 
 ## Custom Project Roles
 
-:::info availability
-Custom project roles were introduced in **Unleash 4.6** and are only available in Unleash Enterprise.
-:::
+:::info availability Custom project roles were introduced in **Unleash 4.6** and are only available in Unleash Enterprise. :::
 
 Custom project roles let you define your own roles with a specific set of project permissions down to the environment level. The roles can then be assigned to users in specific projects. All users have viewer access to all projects and resources, but must be assigned a project role to be allowed to edit a project's resources. For a step-by-step walkthrough of how to create and assign custom project roles, see [_how to create and assign custom project roles_](../how-to/how-to-create-and-assign-custom-project-roles.md).
 
 Each custom project role consists of:
+
 - a **name** (required)
 - a **role description** (optional)
 - a set of **project permissions** (optional)
 - a set of **environment permissions** (optional)
 
+## User Groups
+
+:::info availability User groups are available to Unleash Enterprise users since **Unleash 4.14**. :::
+
+User groups allow you to assign roles to a group of users within a project, rather than a user directly. This allows you to manage your user permissions more easily when there's lots of users in the system. For a guide on how to create and manage user groups see [_how to create and manage user groups_](../how-to/how-to-create-and-manage-user-groups.md).
+
+A user group consists of the following:
+
+- a **name** (required)
+- a **description** (optional)
+- any number of users, but at least one user who is the owner of that group
+
+Groups do nothing on their own, and must be given a role on a project, in same way that a user must be given a role on a project to assign permissions.
```

Comment:
Something like this maybe? I'm not sure this paragraph is necessary, but it _is_ probably a good idea to highlight that groups have no inherent effects 🤔 

By the way is it _on_ or _in_ a project?

```suggestion
Groups do nothing on their own. They must be given a role on a project to assign permissions.
```